### DISCUSSION TITLE
Minting a limited batch at a time?

### DISCUSSION THREAD

**Discussion by Kiaazad** [nftchef/art-engine#21] [Help + Q&A] [answered]
Is it possible to keep a record of generated images, add more parts over time and generate additional images without touching the previously generated ones? If not, may I suggest such a feature please?

**Comment by nftchef** [comment, marked as answer]
there is no built-in support (yet) for continuing generation from a previously generated set. However, this is what the exported _dna.json feature is built to support.

This is not the first time this has been requested 😅, noted! If you need support for this for your project right away, I'd be happy to explain my approach; you can  DM me anytime on twitter or Discord.
